File: BeeCrowd C++/1036.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main() {
    double A, B, C;
    cin >> A >> B >> C;

    double delta = (B * B) - (4 * A * C);

    if (delta < 0 || A == 0) {
        cout << "Impossivel calcular" << endl;
    } else {
        cout << fixed << setprecision(5);

        double r1 = (-B + sqrt(delta)) / (2 * A);
        double r2 = (-B - sqrt(delta)) / (2 * A);

        cout << "R1 = " << r1 << endl;
        cout << "R2 = " << r2 << endl;
    }
    return 0;
}

File: BeeCrowd Python/Iniciante/1036.py
def main():
    a,b,c = input().split( )
    a = float(a)
    b = float(b)
    c = float(c)

    delta = (b**2) - 4 * a * c
    if delta < 0 or a == 0:
        print("Impossivel calcular")
    else:
        r1 = (-b + (delta**0.5)) / (2*a)
        r2 = (-b - (delta**0.5)) / (2*a)

        print(f"""R1 = {r1:.5f}
R2 = {r2:.5f}""")

main()
